spec/requests/editor_spec.rb

```diff
@@ -63,5 +63,25 @@
         expect(response.media_type).to eq("application/json")
       end
     end
+
+    context "with front matter" do
+      it "returns successfully" do
+        sign_in user
+        article_body = <<~MARKDOWN
+          ---
+          ---
+
+          Hello
+        MARKDOWN
+
+        # binding.pry
```

Comment:
probably okay to remove this line.

Reply:
I think it's needed for the specs to pass :joy: